tech/languages/dotnet/about.md

```diff
@@ -4,32 +4,27 @@ subsection: dotnet
 section: tech-languages
 order: 1
 version: 7.8.4
-description: ".NET Core and Mono are open-source .NET platforms. .NET applications are developed using the C#, F# and VB.NET programming languages."
+description: ".NET and Mono are open-source .NET platforms. .NET applications are developed using the C#, F# and VB.NET programming languages."
 permalink: /tech/languages/csharp/dotnet-installation.html
 ---
 
-.NET Core and Mono are available on Fedora. .NET applications are developed using the C#, F# and VB.NET programming languages.
+.NET (and .NET Core) and Mono are available on Fedora. .NET applications are developed using the C#, F# and VB.NET programming languages.
 
-## .NET Core
+## .NET (and .NET Core)
 
-[.NET Core](https://docs.microsoft.com/en-us/dotnet/core/) is a general-purpose, modular, cross-platform and open-source development Platform.
+[.NET](https://docs.microsoft.com/en-us/dotnet/core/) is a general-purpose, modular, cross-platform and open-source development Platform.
 
-.NET Core 3.1 and later versions are packaged in Fedora since 32. Installing the latest SDK can be done with a simple
+.NET 7 and .NET 6 are available in all supported versions of Fedora. Installing an SDK can be done with a simple
 
 ```
-$ sudo dnf install dotnet
+$ sudo dnf install dotnet-sdk-x.y
 ```
 
-For older versions of Fedora and .NET Core 2.1, add the .NET SIG's [copr repository](/deployment/copr/about.html) repository and install the `dotnet-sdk-2.1` package.
+Where `x.y` is a .NET version number, like `7.0` or `6.0`.
 
-```
-$ sudo dnf copr enable @dotnet-sig/dotnet
-$ sudo dnf install dotnet-sdk-2.1
-```
-
-It is not advised to mix these packages with those provided by Microsoft, please disable any other repositories providing dotnet before installing these.
+It is not advised to mix these packages with those provided by Microsoft. Please disable any other repositories providing dotnet before installing these. If you run into any errors that refer to `/usr/share/dotnet` or a missing `libhostfxr.so` see [troubleshoot .NET errors related to missing files on Linux](https://learn.microsoft.com/en-us/dotnet/core/install/linux-package-mixup) which describes how to fix the errors.
 
-See the [.NET Core](dotnetcore.html) page for more information.
+See the [.NET](dotnetcore.html) page for more information.
```

Comment:
This link is broken, even on the live instance at https://developer.fedoraproject.org/tech/languages/csharp/dotnet-installation.html

The link points https://developer.fedoraproject.org/tech/languages/csharp/dotnetcore.html instead of https://developer.fedoraproject.org/tech/languages/dotnet/dotnetcore.html. What's a good way to fix this?

Reply:
2 ways:
1. Delete the following line of YAML at the top of the page
```yaml
permalink: /tech/languages/csharp/dotnet-installation.html
```
This changes the actual link (path) of the HTML file. Not advisable. I see that some links from search engines are pointing to that permalink.

2. adjust the link to a relative path:
```diff
- See the [.NET](dotnetcore.html) page for more information.
+ See the [.NET](../dotnet/dotnetcore.html) page for more information.
```
This one gets resolved correctly.

Reply:
Thanks! Fixed now.